ui/oopsyraidsy/data/manifest.txt

```diff
@@ -11,6 +11,7 @@
 03-hw/dungeon/fractal_continuum.js
 03-hw/dungeon/gubal_library_hard.js
 03-hw/dungeon/sohm_al_hard.js
+03-hw/raid/a12n.js
```

Comment:
The only HW raid.  Foof.  I guess I should have thought about doing these when I did the savages.  I still need to finish off a7s too <_<

Reply:
I was slightly surprised about that too. Not to worry though, a lot of it is reconstructable from logs and recordings!